src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -806,3 +809,18 @@ def __set_errors_json(self, error_count_by_operation, errors_by_operation):
         }
     # endregion
 
+    def __removed_older_complete_status_files(self, status_folder):
+        """ Get the latest status complete file and remove other .complete.status files """
+        removed_files_list = []
+        list_of_files = glob.glob(status_folder + '/' + '*.complete.status')
+
+        for file in list_of_files:
+            if file != self.complete_status_file_path:
+                removed_files_list.append(os.path.basename(file))
```

Comment:
rename this to: files_to_be_removed since there are not really removed at this point

Reply:
done